src/seq_mv/vector.c

```diff
@@ -900,15 +900,9 @@ hypre_SeqVectorElmdivpyMarked( hypre_Vector *x,
 #endif
 
    /* Sanity checks */
-   if (hypre_VectorSize(y) != hypre_VectorSize(b))
+   if (hypre_VectorSize(x) != hypre_VectorSize(b))
    {
-      hypre_error_w_msg(HYPRE_ERROR_GENERIC, "sizes of y and b do not match!\n");
-      return hypre_error_flag;
-   }
-
-   if (hypre_VectorSize(x) < hypre_VectorSize(y))
```

Comment:
Why don't we no longer check for the sizes of x and y?

Reply:
This check (introduced by me last year) was too strict. The function still works if y_size > x_size. So I decided to remove the check